examples/02_meta-analyses/10_plot_cbmr.py

```diff
@@ -0,0 +1,362 @@
+"""
+
+.. _metas_cbmr:
+
+===========================================
+Coordinate-based meta-regression algorithms
+===========================================
+
+A tour of CBMR algorithms in NiMARE
+
+This tutorial is intended to provide a brief description and example of the CBMR
+algorithm implemented in NiMARE. For a more detailed introduction to the elements
+of a coordinate-based meta-regression, see other stuff.
```

Comment:
```suggestion
of a coordinate-based meta-regression, see other stuff.
```

I would link here to the meta analysis course, even if its a draft at this point.

Reply:
Thank you for the suggestions Alejandro! It's a good point, I am considering to link here either a course (like https://www.coursera.org/lecture/functional-mri-2/module-3-meta-analysis-Vd4zz) or a brief overview webpage of this area (like https://libguides.princeton.edu/neuroimaging_meta)